docs/how-to/how-to-run-kata-containers-with-SE-VMs.md

```diff
@@ -0,0 +1,295 @@
+# Kata Containers with IBM Secure Execution VMs
+
+## Manual configuration
+
+This section assumes an environment with a functioning kata container, as per the
+[developer guide](https://github.com/kata-containers/kata-containers/blob/main/docs/Developer-Guide.md).
+
+### Prerequisite
+
+1. Host capable of Secure Execution
+
+To take advantage of the IBM Secure Execution capability, the host machine on which
+you intend to run workloads must be an IBM z15 (or a newer model) or an IBM LinuxONE III
+(or a newer model). In addition to the hardware requirement, you need to verify the CPU
+facility and kernel configuration, as outlined below:
+
+```
+$ # To check the protected virtualization support from kernel
```

Comment:
I think this comment may be misleading. If this returns "found" it simply means the parameter is present. You can, however, still have a kernel which doesn't support SE or a kernel that supports SE, but doesn't have the parameter.
I see two possibilities to make this clearer:
1. If you want to check whether your kernel in theory supports SE, you can check whether `/sys/firmware/uv/prot_virt_host` is present. If it contains `1`, the `prot_virt` parameter was set and the ultravisor was initialized.
1. If you just want to see whether the parameter is there, maybe rephrase the comment to say "Check if the kernel is configured as a SE host"